static/docs/user-guide/glossary.md

```diff
@@ -0,0 +1,30 @@
+# Glossary
```

Comment:
Could this page be generated automatically by the engine instead of having a separate .md file for it? The long descriptions would need to be saved in the glossary.json/js file though...

Reply:
@jorgeorpinel Even I was thinking about the same.